src/uct/ib/dc/dc_mlx5_ep.c

```diff
@@ -1481,30 +1481,20 @@ void uct_dc_mlx5_ep_pending_purge(uct_ep_h tl_ep,
 {
     uct_dc_mlx5_iface_t *iface = ucs_derived_of(tl_ep->iface, uct_dc_mlx5_iface_t);
     uct_dc_mlx5_ep_t *ep       = ucs_derived_of(tl_ep, uct_dc_mlx5_ep_t);
+    uint8_t dci                = ep->dci;
```

Comment:
maybe remove this var?

Reply:
we do `uct_dc_mlx5_iface_dci_detach`, so we should save `dci`